src/homework/day9/ElementsRunner.java

```diff
@@ -0,0 +1,39 @@
+package homework.day9;
+
+import java.util.Arrays;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class ElementsRunner {
+    public static void main(String[] args) {
+//Создать поток данных стрингов elements (Text field, Radio, Check-box, Drop-down, Picker, Breadcrumb) (через Stream.of())
+        Stream<String> streamElements = Stream.of("Text field", "Radio", "Check-box", "Drop-down", "Picker","Breadcrumb");
+//Для каждого элемента стрима пробежаться по отдельным словам и если слово нечетное,
+// то заменить в нем все буквы "e" на букву "o", а если четное, то заменить слово на число количества буков в этом слове
+//Оставить в потоке только уникальные значения
+//Отпечатать в консоль с новой строки
+        streamElements
+                .flatMap(s -> Stream.of(s.split("\\s+")))
+                .map(word -> {
+                    if (word.length() % 2 == 0) {
```

Comment:
Here you are checking if the number of letters in the word is even or odd, but you should check if the position of the word is even or odd.